server/src/main/java/org/opensearch/index/fielddata/IndexFieldData.java

```diff
@@ -192,7 +192,7 @@ public final boolean sortMissingLast(Object missingValue) {
         }
 
         /** Return the missing object value according to the reduced type of the comparator. */
-        public final Object missingObject(Object missingValue, boolean reversed) {
+        public Object missingObject(Object missingValue, boolean reversed) {
```

Comment:
This can result in error? Now that our reduceType() is LONG, it will return Long.MIN_VALUE or Long.MAX_VALUE. Which are not the same for unsigned long.

Reply:
Correct, that's why I needed to drop `final` so I could override this method to take unsigned long into account

Reply:
oh got it.